portal/static/js/accountCreation.js

```diff
@@ -0,0 +1,576 @@
+/*
+ * dependency: JQuery, main.js
```

Comment:
Is there a programmatic way to do that with NPM or a more appropriate framework that makes sense? Should this include i18next (I see it calls `i18next.t()`)?

Reply:
@ivan-c ivan, yeah, my previous comment alluded to a third party modular script loader, or a nodeJS app, which I think follows commonJS module system.